OpenRA.Mods.Common/UtilityCommands/UpgradeModCommand.cs

```diff
@@ -78,42 +77,18 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			// The map cache won't be valid if there was a map format upgrade, so walk the map packages manually
 			// Only upgrade system maps - user maps must be updated manually using --upgrade-map
 			Console.WriteLine("Processing Maps:");
-			foreach (var kv in modData.Manifest.MapFolders)
+			foreach (var map in modData.MapCache.EnumerateMapsWithoutCaching())
 			{
-				var name = kv.Key;
-				var classification = string.IsNullOrEmpty(kv.Value)
-					? MapClassification.Unknown : Enum<MapClassification>.Parse(kv.Value);
-
-				if (classification != MapClassification.System)
-					continue;
-
-				var optional = name.StartsWith("~");
-				if (optional)
-					name = name.Substring(1);
-
 				try
 				{
-					using (var package = (IReadWritePackage)modData.ModFiles.OpenPackage(name))
-					{
-						foreach (var map in package.Contents)
-						{
-							try
-							{
-								using (var mapPackage = package.OpenPackage(map, modData.ModFiles))
-								{
-									if (mapPackage != null)
-										UpgradeMapCommand.UpgradeMap(modData, (IReadWritePackage)mapPackage, engineDate);
-								}
-							}
-							catch (Exception e)
-							{
-								Console.WriteLine("Failed to upgrade map {0}", map);
-								Console.WriteLine("Error was: {0}", e.ToString());
-							}
-						}
-					}
+					Console.WriteLine(map.Package.Name);
```

Comment:
Left over debugging code?

Reply:
This is intentional. We also print out which yaml files are currently upgraded. It also takes quite a while to update all maps, and you're left without knowing if something even happens.